admin/class-parcelpro-shipping.php

```diff
@@ -247,15 +279,44 @@ public function calculate_shipping($package = array())
         }
     }
 
+    /**
+     * @param $rawLastTime string|null The cutoff time of a carrier.
+     *
+     * @return bool Whether the current time is before the cutoff time
+     */
```

Comment:
To me this comment doesn't really add any information that I can't already gather from the function and parameter names. Imo this can either be removed, or we should add more info about what the expected input is (i.e. a time with timezone string, with an example)